app/assets/stylesheets/component-library-overrides.css

```diff
@@ -23,7 +23,14 @@
   }
 }
 .su-underline {
-  --bs-link-decoration: underline solid var(--underline-color) 1px;
+  /* Safari doesn't support one line text-decoration-t
```

Comment:
Is this a typo?  Can you cite a URL so we can track this issue.  This may get fixed at sometime, so we could revert this if we know when that happens.